rust/binding.rs

```diff
@@ -0,0 +1,14 @@
+fn second(f: &Fn()) {
```

Comment:
`_` to suppress compiler warnings about unused variables and avoid giving away the answer.